api/presenter/launch_test.go

```diff
@@ -0,0 +1,56 @@
+package presenter
+
+import (
+	"testing"
+)
+
+func TestLaunch(t *testing.T) {
+	p := LaunchPresenter{
+		ID: 2,
+		LaunchDate: "",
+		LaunchDetails: "",
+		LaunchSuccess: false,
+		VideoLink: "",
+		RocketName: "",
+		MissionName: "",
+	}
+	expect := "{" +
```

Comment:
You can use a [raw string literal ](https://yourbasic.org/golang/multiline-string/)here in order to avoid string concatenation

Reply:
Done